lib/plugins/utils.js

```diff
@@ -52,9 +52,14 @@ export async function loadPlugin({cwd}, name, pluginsPath) {
     ? dirname(resolveFrom.silent(__dirname, pluginsPath[name]) || resolveFrom(cwd, pluginsPath[name]))
     : __dirname;
 
-  // See https://github.com/mysticatea/eslint-plugin-node/issues/250
-  // eslint-disable-next-line node/no-unsupported-features/es-syntax
-  return isFunction(name) ? name : (await import(resolveFrom.silent(basePath, name) || resolveFrom(cwd, name))).default;
+  if (!isFunction(name)) {
+    const file = resolveFrom.silent(basePath, name) || resolveFrom(cwd, name);
+    // See https://github.com/mysticatea/eslint-plugin-node/issues/250
+    // eslint-disable-next-line node/no-unsupported-features/es-syntax
+    name = (await import(`${ file[0] != '.' ? "file://" : "" }${file}`)).default;
```

Comment:
so i dont make incorrect assumptions, could you highlight the details of this change?

Reply:
This change uses the `file://` url schema for import if the file path is not relative (starts with '.').  Node doc indicates this works for import on all platforms.  This is a cheap test instead of using `path.isAbsolute`.  If plugins can be referenced using `http:` schema, this may need to be changed.

Reply:
> This is a cheap test instead of using `path.isAbsolute`.

the benefit of `path.isAbsolute` is that it is self-documenting about why the check is in place. i think i actually prefer that approach here for that reason. any reason not to go that route?

Reply:
> any reason not to go that route?

After looking at the code some more, it appears `file` is always absolute and no check is needed.  I have removed the check so `file://` is always prepended.  Tests under Windows ran fine.

Reply:
this question did go through my mind, but i dismissed it thinking that loading by package name might be the other case. assuming we can get passing builds on this PR, i'll get it this merged.